data/microos/butane/script

```diff
@@ -1,28 +1,71 @@
 #!/bin/bash
-# combustion: network
```

Comment:
Thanks for adding more info. But unfortunately, if the comment is missing the network can't be initialized by combustion.
Maybe commenting this comment would make sense.

Reply:
good point, just accidentally removed. Well, this is written in the docs, I do not find it useful to put redundant information there.

Reply:
Thanks. Then everything is fine.